app/src/renderer/components/common/VmToolBar.vue

```diff
@@ -1,25 +1,21 @@
 <template lang="pug">
-  tm-tool-bar
+  .tm-tool-bar
     a.back(@click="back" :disabled="user.history.length === 0" v-tooltip.bottom="'Back'")
       i.material-icons arrow_back
     slot
     a.help(@click="enableModalHelp" v-tooltip.bottom="'Help'")
       i.material-icons help_outline
-    router-link.settings(to="/preferences" v-tooltip.bottom="'Preferences'")
-      i.material-icons#settings settings
+    a#settings(@click="goToPreferences" v-tooltip.bottom="'Preferences'")
```

Comment:
is there a reason we're not using router-link? click event with a method on an `a` tag seems strange.

Reply:
yeah maybe, it was one of my desperate attempts to fix the e2e tests lol. Feel free to push changes

Reply:
I'll update it for an `href`

Reply:
reverted changes